app/src/components/market/common/market_top_details_closed/index.tsx

```diff
@@ -49,7 +49,7 @@ const MarketTopDetailsClosed: React.FC<Props> = (props: Props) => {
     address,
     answerFinalizedTimestamp,
     arbitrator,
-    collateral: collateralToken,
+    collateral,
```

Comment:
Thanks for fixing this :)
This keeps bugging me when I come to this file because in every other component collateral is the actual token but in here it is the address of the token.